homeassistant/components/plex/config_flow.py

```diff
@@ -225,6 +230,43 @@ def __init__(self):
         _LOGGER.debug("Imported Plex configuration")
         return await self.async_step_server_validate(import_config)
 
+    async def async_step_plex_website_auth(self):
+        """Begin external auth flow on Plex website."""
+        payload = {
+            "X-Plex-Device-Name": X_PLEX_DEVICE_NAME,
+            "X-Plex-Version": X_PLEX_VERSION,
+            "X-Plex-Product": X_PLEX_PRODUCT,
+            "X-Plex-Device": self.hass.config.location_name,
+            "X-Plex-Platform": X_PLEX_PLATFORM,
+            "X-Plex-Model": "Plex OAuth",
+        }
+        self.plexauth = PlexAuth(payload)
```

Comment:
Please have the client accept an optional aiohttp session. We want all integrations that use aiohttp client interface to use the same session that home assistant creates.

Use the aiohttp client helper to get the home assistant session.

https://github.com/home-assistant/home-assistant/blob/454d479b36b314e3d2308f8ef8571ae38cafa4ad/homeassistant/helpers/aiohttp_client.py#L31

Reply:
Bumped library and the hass session is now passed in.